packages/react-composites/tests/browser/utils.ts

```diff
@@ -49,10 +72,35 @@ export const createUserAndThread = async (
     endpointUrl,
     displayName,
     threadId,
-    topic
+    topic: TOPIC_NAME
   }));
 };
 
-export const getNameInitials = (name: string): string => {
-  return Array.prototype.map.call(name.split(' '), (x) => x.substring(0, 1).toUpperCase()).join('');
+export const loadPage = async (browser: Browser, serverUrl: string, user: IdentityType): Promise<Page> => {
+  const qs = encodeQueryData(user);
+  const page = await browser.newPage();
+  await page.setViewportSize(PAGE_VIEWPORT);
+  const url = `${serverUrl}?${qs}`;
+  console.log(`Loading Chat app for ${user.displayName} at ${url}`);
+  await page.goto(url, { waitUntil: 'networkidle' });
+  // Important: For ensuring that blinking cursor doesn't get captured in
+  // snapshots and cause a diff in subsequent tests.
+  page.addStyleTag({ content: `* { caret-color: transparent !important; }` });
+  return page;
+};
+
+const encodeQueryData = (data: IdentityType): string => {
+  const qs: Array<string> = [];
+  for (const d in data) {
+    qs.push(encodeURIComponent(d) + '=' + encodeURIComponent(data[d]));
+  }
+  return qs.join('&');
+};
+
+const CONNECTION_STRING = process.env.CONNECTION_STRING ?? '';
```

Comment:
nit: same as before, better to have those constants declared at the beginning

Reply:
+1